pkg/Microsoft.Private.CoreFx.UAP/Microsoft.Private.CoreFx.UAP.pkgproj

```diff
@@ -1,6 +1,7 @@
 ﻿<Project DefaultTargets="Build">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Directory.Build.props))\Directory.Build.props" />
   <PropertyGroup>
+    <SkipValidatePackage>true</SkipValidatePackage>
```

Comment:
This needs a comment and an issue.

Reply:
I clarified offline, we should remove the transport package. You decide if you want to do it here or separately.

Reply:
I will remove it from 5.0 but do we have to remove it from 3.1 branch as well ?

Reply:
Yeah, let's remove it here too.

Reply:
Removing it in the 3.1 branch as well is desired as that will be our LTSB.